tests/containers/integration-test-base

```diff
@@ -8,3 +8,9 @@ RUN dnf upgrade --refresh -y --nodocs && \
         systemd-devel \
     -y && \
     dnf -y clean all
+
+# First, install dasbus via dnf (v1.4) so the package can be provided.
+# Then upgrade dasbus via pip to v1.7 (required by python3-bluechi)
+RUN dnf install -y python3-dasbus
```

Comment:
Hmm, do we really require python3-dasbus >= 1.7 (feature/bug wise)? If so, then we should work with RHEL maintainers to try to do a version bump in CS9

Reply:
Also if we don't really require 1.7, I'd downgrade the requirement in PIP package to require only 1.4+

Reply:
We need it feature-wise, unfortunately. Reaching out to the maintainers seems to be a longer process - can we keep it like this for now and create an issue for updating the package in CS9 as well as changing it here?

Reply:
Update: see https://github.com/containers/bluechi/pull/485#discussion_r1301075015